npm/napi/src/lib.rs

```diff
@@ -0,0 +1,233 @@
+#![deny(clippy::all)]
```

Comment:
Please add the license line (on this and other source files).

Probably same as https://github.com/denoland/deno/blob/main/ext/kv/01_db.ts#L1

Reply:
done, added to all non-generated source files